### PR TITLE
@metamask/eslint-config*@6.0.0

### PR DESCRIPTION
**Pull request by rekmarks**
- Add/update:
  - `@metamask/eslint-config@6.0.0`
  - `@metamask/eslint-config-mocha@6.0.0`
  - `@metamask/eslint-config-nodejs@6.0.0`
- Update `.eslintrc.js` per new configs (base config includes prettier)
- `yarn lint:fix`

Clean run:
> ✨  Done in 88.49s.

Cached run:
> ✨  Done in 5.27s.

😎

### CODE REVIEW

File: ui/app/components/app/asset-list-item/asset-list-item.js [hunk 1]
```diff
@@ -131,27 +131,27 @@ const AssetListItem = ({
...
-  onClick: PropTypes.func.isRequired,
-  tokenAddress: PropTypes.string,
-  tokenSymbol: PropTypes.string,
-  tokenDecimals: PropTypes.number,
-  tokenImage: PropTypes.string,
-  warning: PropTypes.node,
-  primary: PropTypes.string,
-  secondary: PropTypes.string,
-  identiconBorder: PropTypes.bool,
+  'iconClassName': PropTypes.string,
```
Comment (brad-decker) [inline ui/app/components/app/asset-list-item/asset-list-item.js:136]: is it too late for me to hate this? 😆

**Review by brad-decker — APPROVED** [review]
other than the rule thing I commented on this looks dope. I can learn to not hate the rule, if it changes it for me on save 😂

File: ui/app/components/app/asset-list-item/asset-list-item.js (hunk 1 above)
Comment (rekmarks) [reply ui/app/components/app/asset-list-item/asset-list-item.js:136]: Definitively yes! 😂

Comment (rekmarks) [reply ui/app/components/app/asset-list-item/asset-list-item.js:136]: We're accepting PRs over at https://github.com/MetaMask/eslint-config

**Pull request merged**